lib/noisecancellationadapter.ts

```diff
@@ -38,9 +38,16 @@ class NoiseCancellationAdapter  {
   async init(sdkRootPath: string, sdkFile: string) : Promise<AudioProcessor> {
     try {
       this._log.debug('loading noise cancellation sdk: ', sdkRootPath);
-      const module = await import(/* webpackIgnore: true */ `${sdkRootPath}/${sdkFile}`);
-      this._log.debug('Loaded noise cancellation sdk:', module);
-      const sdkAPI = module.default as NoiseCancellationSDK;
+
+      // NOTE(mpatwardhan): Typescript has an issue - when module set to "commonjs"
+      // typescript converts dynamic import calls to require :(
+      // https://github.com/microsoft/TypeScript/issues/43329
+      // const dynamicModule = await import(/* webpackIgnore: true */ `${sdkRootPath}/${sdkFile}`);
+      // to workaround this issue we move import call into seperate modue (dynamicImport) that is not
+      // compiled with typescript.
```

Comment:
I don't think we need to add this comment block here. Can we just keep it inside the dynamicImport  module?